src/sentry/snuba/discover.py

```diff
@@ -681,6 +682,14 @@ def top_events_timeseries(
             referrer=referrer,
         )
 
+    if not allow_empty and not result.get("data", []):
```

Comment:
nit: Should we use `len` here instead of treating `[]` as falsey

Reply:
fixed in 10f9070